assets/maps/prefabs/space/prefab_hunting.dmm

```diff
@@ -0,0 +1,502 @@
+//MAP CONVERTED BY dmm2tgm.py THIS HEADER COMMENT PREVENTS RECONVERSION, DO NOT REMOVE
+"a" = (
+/obj/machinery/door/airlock/pyro/external{
+	dir = 4
+	},
+/turf/simulated/floor/dojo,
+/area/prefab/huntingsupplies)
+"b" = (
+/obj/table/reinforced,
+/obj/item/plate{
+	pixel_y = 7
+	},
+/obj/item/reagent_containers/food/snacks/ingredient/meat/fish/fillet_slice/salmon{
+	pixel_y = 7;
+	pixel_x = 2
+	},
+/turf/simulated/floor/wood/three,
+/area/prefab/huntingsupplies)
+"c" = (
+/obj/decal/stripe_delivery,
+/obj/table/reinforced/industrial/auto,
+/obj/item/reagent_containers/glass/oilcan{
+	pixel_y = 12
+	},
+/obj/item/reagent_containers/food/drinks/fueltank{
+	pixel_y = 4;
+	pixel_x = -4;
+	layer = 3.01
+	},
+/turf/simulated/floor/dojo,
+/area/prefab/huntingsupplies)
+"d" = (
+/obj/rack,
+/obj/item/kitchen/utensil/knife{
+	pixel_x = -4
+	},
+/obj/item/kitchen/utensil/knife{
+	pixel_x = 4
+	},
+/turf/simulated/floor/wood/three,
+/area/prefab/huntingsupplies)
+"e" = (
+/obj/machinery/light/small/floor/netural,
+/turf/simulated/floor/engine/caution/east,
+/area/prefab/huntingsupplies)
+"f" = (
+/obj/rack,
+/obj/item/ammo/bullets/a12/bird,
+/turf/simulated/floor/wood/three,
+/area/prefab/huntingsupplies)
+"h" = (
+/obj/machinery/door/airlock/pyro{
+	requires_power = 0;
+	dir = 8
+	},
+/obj/mapping_helper/airlock/bolter,
+/turf/simulated/floor/dojo,
+/area/prefab/huntingsupplies)
+"k" = (
+/obj/stool/chair/green,
+/mob/living/carbon/human/normal/assistant{
```

Comment:
If you're going to spawn mob humans, I'd code actual entities with unique designs and not just staff assistants.

Additionally, currently they will have an SSD message and it's not inherently obvious that they're NPCs.

Reply:
i don't really know how to code...